polkadot/runtime/kusama/src/lib.rs

```diff
@@ -157,11 +158,16 @@ pub fn native_version() -> NativeVersion {
 	NativeVersion { runtime_version: VERSION, can_author_with: Default::default() }
 }
 
-/// We currently allow all calls.
-pub struct BaseFilter;
-impl Contains<RuntimeCall> for BaseFilter {
-	fn contains(_c: &RuntimeCall) -> bool {
-		true
+/// Disable all calls to the Identity pallet. This will lock the state of the pallet, preventing
+/// further updates to identities and sub-identities. The locked state will be the genesis state of
+/// a new system chain and then removed from the Relay Chain.
+pub struct IdentityCalls;
+impl Contains<RuntimeCall> for IdentityCalls {
+	fn contains(c: &RuntimeCall) -> bool {
+		match c {
+			RuntimeCall::Identity(_) => true,
```

Comment:
This looks a bit weird, as the comment states that it's disabling all calls to the identity pallet, yet here it's returning true only for calls to the identity pallet.